### PR TITLE
File Added : Assignment 4 & 5

### PR DESCRIPTION
**Pull request by urmilagagare**

### CODE REVIEW

File: urmila/JAVASCRIPT/Assignment4/Automorphic.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (shubhamdalal49) [inline urmila/JAVASCRIPT/Assignment4/Automorphic.html:3]: check indentation

File: urmila/JAVASCRIPT/Assignment4/Automorphic.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        .jumbotron {
+            border-radius: 10%;
+        }
+        input[type="number"],
+        select {
+            border-radius: 10px;
+        }
+    </style>
+</head>
+        <div class="jumbotron bg-white bord" style="background-image: linear-gradient(120deg, #15d6ac, #c5dad4)">
```
Comment (shubhamdalal49) [inline urmila/JAVASCRIPT/Assignment4/Automorphic.html]: body tag is missing

File: urmila/JAVASCRIPT/Assignment4/Krishnamurthy.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (shubhamdalal49) [inline urmila/JAVASCRIPT/Assignment4/Krishnamurthy.html]: check indentation

File: urmila/JAVASCRIPT/Assignment4/Krishnamurthy.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        .jumbotron {
+            border-radius: 10%;
+        }
+        input[type="number"],
+        select {
+            border-radius: 10px;
+        }
+    </style>
+</head>
+        <br /><br />
```
Comment (shubhamdalal49) [inline urmila/JAVASCRIPT/Assignment4/Krishnamurthy.html]: missing body tag

File: urmila/JAVASCRIPT/Assignment5/fibonaccies.html [hunk 5]
```diff
@@ -0,0 +1,61 @@
+<html lang="en">
+
```
Comment (shubhamdalal49) [inline urmila/JAVASCRIPT/Assignment5/fibonaccies.html]: remove extra line

File: urmila/JAVASCRIPT/Assignment5/fibonaccies.html [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+                c = a + b;
+                a = b;
+                b = c;
+                document.write(" " + c + " ");
+            }
+        }
+    </script>
+</head>
+
+<body>
```
Comment (shubhamdalal49) [inline urmila/JAVASCRIPT/Assignment5/fibonaccies.html]: check indentation and remove extra line

File: urmila/JAVASCRIPT/Assignment5/fibonaccies.html [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+        <h3 id="p"></h3>
+        <p id="pid"></p>
+        <p id="pid1"></p>
+        <br>
+    </div>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.bundle.min.js"integrity="sha384-Piv4xVNRyMGpqkS2by6br4gNJ7DXjqk09RmUpJ8jgGtD7zP9yug3goQfGII0yAns"crossorigin="anonymous"></script>
+    </div>
+</body>
+</html>
```
Comment (shubhamdalal49) [inline urmila/JAVASCRIPT/Assignment5/fibonaccies.html]: add extra line at the end and check indentation

File: urmila/JAVASCRIPT/Assignment5/prime.html [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+                position: relative;
+                background-color:red;
+                border: none;
+                font-size: 20px;
+                color: black;
+                padding: 10px;
+                width: 180px;
+                text-align: center;
+                -webkit-transition-duration: 0.4s;
+                transition-duration: 0.4s;
```
Comment (shubhamdalal49) [inline urmila/JAVASCRIPT/Assignment5/prime.html]: style tag should be in head tag

File: urmila/JAVASCRIPT/Assignment5/prime.html [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+        <div class="card">
+     <div class="container">
+        <h2> To print Prime numbers between two integers!</h2>
+        Start:  <input type="number" name="start" id="start" min="1" style="width: 100px;" value="10" />&nbsp;
+        End:  <input type="number" name="end" id="end" min="1" style="width: 100px;" value="100" />&nbsp;<input type="submit" value="Print Prime Numbers" onclick="printPrime()" name="print" />
+        <center><div id="result"></div></center>
+    </div>
+        </div>
+    </body>
+</html>
```
Comment (shubhamdalal49) [inline urmila/JAVASCRIPT/Assignment5/prime.html]: add extra line at the end

File: urmila/JAVASCRIPT/Assignment5/sum_number.html [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+            <p id="pid"></p>
+            <p id="pid1"></p>
+            <br>
+            </div>
+            <script
+                src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous" ></script>
+                <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.bundle.min.js"integrity="sha384-Piv4xVNRyMGpqkS2by6br4gNJ7DXjqk09RmUpJ8jgGtD7zP9yug3goQfGII0yAns"crossorigin="anonymous" ></script>
+        </div>
+    </body>
+</html>
```
Comment (shubhamdalal49) [inline urmila/JAVASCRIPT/Assignment5/sum_number.html]: add extra line at the end

File: urmila/JAVASCRIPT/Assignment5/triangle.html [hunk 11]
```diff
@@ -0,0 +1,61 @@
+<html lang="en">
+<head>
```
Comment (shubhamdalal49) [inline urmila/JAVASCRIPT/Assignment5/triangle.html]: check indentation

File: urmila/JAVASCRIPT/Assignment5/triangle.html [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+            integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
+            crossorigin="anonymous">
+        </script>
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.bundle.min.js"
+            integrity="sha384-Piv4xVNRyMGpqkS2by6br4gNJ7DXjqk09RmUpJ8jgGtD7zP9yug3goQfGII0yAns"
+            crossorigin="anonymous">
+        </script>
+        </div>
+    </body>
+</html>
```
Comment (shubhamdalal49) [inline urmila/JAVASCRIPT/Assignment5/triangle.html]: add extra line at the end

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

File: urmila/JAVASCRIPT/Assignment4/Automorphic.html [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+            <center>
+                <p class="bg-success" id="pid"></p>
+            </center>
+        </div>
+        </div>
+        <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
+        <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"crossorigin="anonymous"></script>
+        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"crossorigin="anonymous"></script>
+    </body>
+</html>
```
Comment (Pawan1005) [inline urmila/JAVASCRIPT/Assignment4/Automorphic.html:77]: check indentation

File: urmila/JAVASCRIPT/Assignment4/Krishnamurthy.html [hunk 14]
```diff
@@ -0,0 +1,75 @@
...
+            .jumbotron {
+                border-radius: 10%;
+            }
+            input[type="number"],
+            select {
+                border-radius: 10px;
+            }
+        </style>
+    </head>
+    <br /><br />
```
Comment (Pawan1005) [inline urmila/JAVASCRIPT/Assignment4/Krishnamurthy.html]: remove br tag

File: urmila/JAVASCRIPT/Assignment4/upper_lower.html [hunk 15]
```diff
@@ -0,0 +1,69 @@
...
+                cursor: pointer;
+                margin-left: 130px;
+            }
+            #pid{
+                color: black;
+                text-align: center;
+                font-size: large;
+            }
+        </style>
+         <script>
```
Comment (Pawan1005) [inline urmila/JAVASCRIPT/Assignment4/upper_lower.html]: one extra space added

File: urmila/JAVASCRIPT/Assignment5/fibonaccies.html [hunk 16]
```diff
@@ -0,0 +1,59 @@
...
+            <h3 id="p"></h3>
+            <p id="pid"></p>
+            <p id="pid1"></p>
+            <br>
+        </div>
+        <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.bundle.min.js"integrity="sha384-Piv4xVNRyMGpqkS2by6br4gNJ7DXjqk09RmUpJ8jgGtD7zP9yug3goQfGII0yAns"crossorigin="anonymous"></script>
+        </div>
+    </body>
+</html>
```
Comment (Pawan1005) [inline urmila/JAVASCRIPT/Assignment5/fibonaccies.html:57]: check indentation

File: urmila/JAVASCRIPT/Assignment5/sum.html [hunk 17]
```diff
@@ -0,0 +1,75 @@
...
+            .jumbotron {
+                border-radius: 10%;
+            }
+            input[type="number"],
+            select {
+                border-radius: 10px;
+            }
+        </style>
+    </head>
+            <br><br>
```
Comment (Pawan1005) [inline urmila/JAVASCRIPT/Assignment5/sum.html]: body tag missing

File: urmila/JAVASCRIPT/Assignment5/sum.html [hunk 18]
```diff
@@ -0,0 +1,75 @@
...
+            <div class="jumbotron bg-white bord" style="background-image: linear-gradient(120deg, #15d6ac, #c5dad4)">
+                <center>
+                    <h2>Sum of edge equal center</h2>
+                    <hr />
+                </center>
+
```
Comment (Pawan1005) [inline urmila/JAVASCRIPT/Assignment5/sum.html]: remove extra line

File: urmila/JAVASCRIPT/Assignment5/triangle.html [hunk 19]
```diff
@@ -0,0 +1,54 @@
...
+            <h3 id="p"></h3>
+            <p id="pid"></p>
+            <p id="pid1"></p>
+            <br>
+        </div>
+        <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.bundle.min.js" integrity="sha384-Piv4xVNRyMGpqkS2by6br4gNJ7DXjqk09RmUpJ8jgGtD7zP9yug3goQfGII0yAns"crossorigin="anonymous"> </script>
+        </div>
+    </body>
+</html>
```
Comment (Pawan1005) [inline urmila/JAVASCRIPT/Assignment5/triangle.html:56]: check indentation

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**